src/internal/connector/onedrive/collections.go

```diff
@@ -352,23 +357,31 @@ func (c *Collections) Get(
 				c.source)
 			if err != nil {
 				return nil, nil,
-					clues.Wrap(err, "making exclude prefix for drive").WithClues(ctx).With("drive_id", driveID)
+					clues.Wrap(err, "making exclude prefix").WithClues(ctx)
```

Comment:
so many context switches ;)
```suggestion
					clues.Wrap(err, "making exclude prefix").WithClues(ictx)
```

Reply:
so many rebase collisions! 🎉 

(thanks for the catch)